src/chrome/content/rules/Red_Hat.xml

```diff
@@ -168,10 +160,10 @@
 		<test url="http://et.redhat.com/foo?bar" />
 
 	<rule from="^http://investors\.redhat\.com/"
-		to="https://investor.shareholder.com/" />
+		to="https://investors.redhat.com/" />
```

Comment:
Thanks for the feedback! I cleaned up these and removed both jobs and investors from the 'complications' section.